teams/teams-ps/teams/Set-CsTeamsCallingPolicy.md

```diff
@@ -557,6 +559,44 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -EnableSpendLimits
+This setting allows an admin to enable or disable spend limits on PSTN calls for their user base.
+
+Possible values: 
+- True
+- False
+
+```yaml
+Type: Boolean
+Parameter Sets: (All)
+Aliases:
+Applicable: Microsoft Teams
+
+Required: False
+Position: Named
+Default value: None
```

Comment:
Please add the default value

Reply:
Updated. Added 'False' as the default value.